### ISSUE TITLE
508-defect-2 [SCREENREADER]: Incomplete form submission, error message vague and not read assertively

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va-forms-system-core#379]
Label: 24069

    - [508-defect-2]
     
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2

Issue Title: 508 Defect 2, screen readers, incomplete form submission, error message vague and not read assertively.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, I expect error messages to be read assertively and focus placed at the point of the error. As a user, I expect error messages to inform me as to the nature of the error and offer a solution.
<h2>Details</h2>
<p>When I change the board review option to "hearing" and do not specify a hearing type, the error message when I attempt to submit is not spoken assertively by my screen reader. I have to arrow around the screen to locate it. </p>
<p> The error message says that information is missing, but does not specify which information. This increases cognitive load and could cause confusion for users with cognitive disabilities. </p>
<h2>Acceptance Criteria</h2>
Error messages are informative and spoken assertively.
<h2>Environment</h2>
When the error is generated, NVDA announces "alert," while JAWS says nothing.
<h2>Steps to Recreate</h2>
<ul>
<li>Access 

https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182

and sign in with any test user. </li>
<li>Complete the form, initially stating you will upload evidence.</li>
<li>On the review screen, change the board review option to "hearing with a veterans law judge, do not specify the hearing type.</li>
<li>Attempt to submit</li>
</ul> 

![image](https://user-images.githubusercontent.com/78870095/119361933-095ef600-bc61-11eb-9272-d4dc35b7f64b.png)

**Comment by obliviga** [comment]
Verified on prod: ![image.png](https://images.zenhubusercontent.com/6283eaffdea4c21364d67e5a/7cb6ab02-ccc1-4482-afed-42d55317e07d)
